FeedStoreChallenge/CoreDataFeed.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  CoreDataFeed+CoreDataProperties.swift
+//  FeedStoreChallenge
+//
+//  Created by Alex Thurston on 5/7/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+//
+
+import Foundation
+import CoreData
+
+@objc(CoreDataFeed)
+class CoreDataFeed: NSManagedObject {
+	@NSManaged public var timestamp: Date
+	@NSManaged public var feedImages: NSOrderedSet
+}
```

Comment:
`CoreDataFeed` is `internal`. No need to mark properties as `public`.
Also `feedImages` could be renamed to `images` since the context is clear here. The `CoreDataFeed` class already represents a `feed` so you avoid duplication like `feed.feedImages` as `feed.images`.

Please remove `public` access modifiers and rename `feedImages` to `images`.

```suggestion
class CoreDataFeed: NSManagedObject {
	@NSManaged var timestamp: Date
	@NSManaged var images: NSOrderedSet
}
```